site/docs/v1/tech/apis/identity-providers/xbox.adoc

```diff
@@ -17,6 +17,8 @@ The Xbox identity provider type will use the Xbox OAuth v2.0 login API. It will
 
 This identity provider will call Xbox’s API to load the user's `email` and `gtg` (Gamer Tag) and use those as `email` and `username` to lookup or create a user in FusionAuth depending on the linking strategy configured for this identity provider. Additional claims returned by Xbox can be used to reconcile the user to FusionAuth by using an Xbox Reconcile Lambda.
 
+:idp_long_lived_token_type: refresh
+:idp_refresh_additional_info_url: https://learn.microsoft.com/en-us/gaming/gdk/_content/gc/live/features/s2s-auth-calls/service-authentication/live-service-authentication-nav
```

Comment:
We could add a disclaimer that you have to have a developer access 
<img width="953" alt="image" src="https://github.com/FusionAuth/fusionauth-site/assets/16090626/66a4e9d7-5423-4224-a260-b20e924dcbed">